country.go

```diff
@@ -2038,6 +2060,12 @@ func ByNumeric(num int) (Info, bool) {
 	return code, code.ISO3166_1_numeric != 0
 }
 
+// IsEuropean returns true if the given ISO2 or ISO3 country code is members of the European Union (EU).
+func IsEuropean(countryCode string) bool {
```

Comment:
Maybe just to clarify, the naming should include EuropeanUnion, not just European